net/gatling/Makefile

```diff
@@ -35,6 +35,12 @@ endef
 define Package/gatling/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/gatling $(1)/usr/bin/
+
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_CONF) ./files/gatling.conf $(1)/etc/config/gatling
```

Comment:
Shouldnt this be marked as conffile?

Reply:
The file is a regular uci configuration file, that gets copied to `/etc/config/`. Do you have something specific in mind for marking this file?

I thought, using the `$(INSTALL_CONF)` macro should be sufficient.

Reply:
That macro sets correct permissions as it is said here: https://openwrt.org/docs/guide-developer/packages

``conffiles``, on the other hands, prevents that the config is overwritten by updating this package.
- You can test on your router: modify config, reinstall it and config file was replaced by the version from the package.

Reply:
Ah, I see. Thank you very much for the hint. I added that section in the current version of the PR.